spec/lib/publishing_api_consumer_spec.rb

```diff
@@ -8,24 +8,26 @@
   it_behaves_like 'a message queue processor'
 
   context 'when the Dimensions::Item already exists - all events but unpublish' do
-    let(:latest_item) { create(:dimensions_item, latest: true, outdated: false) }
+    let!(:latest_item_en) { create(:dimensions_item, latest: true, outdated: false, locale: 'en') }
+    let!(:latest_item_de) { create(:dimensions_item, latest: true, outdated: false, locale: 'de') }
```

Comment:
I would make it implicit that all items are `latest`, `outdated` and `en` in the Factory. And only set values if this is not the case. This would improve readability.